spec/dictionary_spec.rb

```diff
@@ -1,2 +1,20 @@
-require "./lib/scrabble_master.rb"
+require "spec_helper"
 
+describe "Dictionary" do
+  before :each do
+    @dictionary = Scrabble::Dictionary.new
+  end
+
+  context "initializing" do
+    it "returns a dictionary object" do
+      expect(@dictionary).to be_an_instance_of Scrabble::Dictionary
+    end
+  end
+  context "returns words from dictionary" do
+    it "returns valid words only" do
+      expect(Scrabble::Dictionary.valid_word?("ABACAS")).to be_truthy
+      expect(Scrabble::Dictionary.valid_word?("abamp")).to be_truthy
+      expect(Scrabble::Dictionary.valid_word?("flfflflfl")).to eq false
```

Comment:
It'd be good to have a test case where the "word" is not itself a word, but is a subset of a valid word.